cmd/config.go

```diff
@@ -369,8 +364,9 @@ func getDefaultData(fs vfs.FS) (map[string]interface{}, error) {
 	group, err := user.LookupGroupId(currentUser.Gid)
 	if err == nil {
 		data["group"] = group.Name
-	} else if cgoEnabled {
-		// Only return an error if CGO is enabled.
+	} else if cgoEnabled && runtime.GOOS != "windows" {
```

Comment:
I've removed nearly all of the uses of `runtime.GOOS` that I added.  I left this one in because I really think it makes more sense to just add this boolean check rather than try to refactor this into separate files.